tuta-sdk/rust/src/entities/mod.rs

```diff
@@ -15,7 +15,7 @@ use crate::date::DateTime;
 use crate::generated_id::GeneratedId;
 use crate::custom_id::CustomId;
 
-pub trait Entity {
+pub trait Entity: 'static {
```

Comment:
Paul helped me solve the mocking with it. He said it is fine because it's on trait bound: https://doc.rust-lang.org/rust-by-example/scope/lifetime/static_lifetime.html#trait-bound

Reply:
ah cool I didn't know how it works!
Can we maybe add a comment explaining this and also maybe this link?